monkey/common/user_roles.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class UserRoles(Enum):
+    """
+    An Enum representing user roles.
+    This Enum represents roles that the user can have. The value
+    of each member is the description of the role
+    """
+
+    ISLAND = "Monkey Island, C&C Server"
```

Comment:
This is a poor description. C&C server is not a role. Also we need to distinguish the terms "user" that means an account vs "user" that "uses" the island for example.

Reply:
Suggest something

Reply:
Suggested in a comment above. Also, Island is not a role. Island is the server that accounts authenticate to.